website/payments/templates/admin/payments/batch_topic.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'admin/base_site.html' %}
+{% load i18n admin_urls static admin_modify %}
+
+{% block title %}{% trans "Export payment batch topic" %} | {{ event.title }} | {{ site_title|default:_('Thalia site admin') }}{% endblock %}
+{% block opengraph_title %}{% trans "Export payment batch topic" %} | {{ event.title }} | {{ site_title|default:_('Thalia site admin') }}{% endblock %}
+
+{% block branding %}
+<h1 id="site-name"><a href="{% url 'admin:index' %}">{{ site_header|default:_('Thalia administration') }}</a></h1>
+{% endblock %}
+
+{% block content_title %}<h1>{% trans "Topics of payment batch:" %}</h1>{% endblock %}
+{% block content %}
+<div id="content-main">
+    <textarea onclick="this.select()" readonly="readonly" cols="80" rows="10">{{ description }}</textarea>
```

Comment:
If this is supposed to be copied, then why not just let the code download a txt file?

Reply:
Could be. I do think this works faster because you're not filling up your downloads folder, but that's a real personal preference.